content/docs/tutorials/interactivity.md

```diff
@@ -0,0 +1,26 @@
+---
+$title: Create interactive AMP pages
+$order: 1
+---
+
+In this tutorial, you'll learn how to use data binding and expressions to build awesome, interactive AMP pages with [`<amp-bind>`](https://www.ampproject.org/docs/reference/components/amp-bind.html).
+
+<div class="img-right">
+<amp-img  src="/static/img/docs/tutorials/interactivity-final-page.png"
+         width="415" height="738" layout="responsive"
+         alt="Sample e-commerce product page">
+</amp-img>
+</div>
+
+## What you'll build
+
+You'll build an e-commerce product page that:
+
+- Uses AMP HTML and AMP components for a fast and rich user experience.
+- Includes cross-element interactivity by using `<amp-bind>`.
```

Comment:
I'd link <amp-bind> in a few more places. Just a nit, but I think the links don't hurt.